Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay

```diff
@@ -30,44 +30,45 @@ current_time=$(eval $time)
 present_dir=$(pwd)
 echo $present_dir
 
-bash Openshift-EE/utils/pooling jobname: nfs-app-kill
-bash Openshift-EE/utils/e2e-cr jobname:jiva-controller-network-delay jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+#Creating e2e custom resource result for all the tests running in this stage.
+#In Gitlab, all the jobs in a stage runs parallely. Inorder to run this serially, each job in the stage waits for the other to get completed.
 
-echo "*******Performing network delay on controller**********"
+bash Openshift-EE/utils/pooling jobname:jiva-controller-kill
+bash Openshift-EE/utils/e2e-cr jobname:jiva-controller-network-delay jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
 
+echo "*******Performing network delay on Target**********"
+## testcase name is defined in test_vars of LitmusBook.
 run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-target-network-delay metadata=${run_id})
 echo $test_name
 
 cd litmus
-cp experiments/chaos/openebs_target_network_delay/run_litmus_test.yml run_controller_delay_test_jiva.yml
+# copy the content of application pod failure chaos `run_litmus_test.yml` into a different file to update the test specific parameters.
+cp experiments/chaos/openebs_target_network_delay/run_litmus_test.yml run_targetdelay_test.yml
```

Comment:
@w3aman use the file name like 'run_target_delay_test.yml`

Reply:
Okay!